javav2/usecases/creating_lambda_ppe/src/main/java/com/example/ppe/DynamoDBService.java

```diff
@@ -0,0 +1,82 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.ppe;
+
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
+import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
+import software.amazon.awssdk.services.dynamodb.model.DynamoDbException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class DynamoDBService {
+
+    private DynamoDbClient getClient() {
+
+        // Create a DynamoDbClient object.
+        Region region = Region.US_EAST_1;
+        DynamoDbClient ddb = DynamoDbClient.builder()
+                .region(region)
+                .build();
+
+        return ddb;
+    }
+
+    // Persist the PPE Items into the Gear table.
+    public void persistItem(List<ArrayList<GearItem>> gearList) {
+        DynamoDbClient ddb = getClient();
+
+        try {
+
+            DynamoDbEnhancedClient enhancedClient = DynamoDbEnhancedClient.builder()
+                    .dynamoDbClient(ddb)
+                    .build();
+
+            DynamoDbTable<Gear> gearTable = enhancedClient.table("Gear", TableSchema.fromBean(Gear.class));
+            Gear gearRecord;
+
+            // Create an Instant.

```

Comment:
Instance?

Reply:
Made the following fix - "Create an Instant object"